integration-tests/oidc/src/test/java/io/quarkus/it/keycloak/CodeFlowInGraalITCase.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.it.keycloak;
+
+import org.junit.jupiter.api.Disabled;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.SubstrateTest;
+
+/**
+ * @author <a href="mailto:psilva@redhat.com">Pedro Igor</a>
+ */
+@QuarkusTestResource(KeycloakTestResource.class)
+@SubstrateTest
+@Disabled("While figuring out how to have different application.properties for different tests")
```

Comment:
This is only possible for native tests at the moment by creating new maven modules

Reply:
Actually it looks like this only really needs to set runtime config? If so it may be possible to use multiple maven profiles, and tell surefire/failsafe to use a different profile for each run.

Reply:
I'll try what you suggest in regard to using profiles. But, wouldn't be more simple if we could annotate a test or reuse some test annotation to specify the properties we want to load ?

Reply:
@stuartwdouglas I've added a new module for code flow tests.